athenzdomains.go

```diff
@@ -0,0 +1,183 @@
+package webhook
```

Comment:
Let's also rename the athenzdomains.go file to zpeclient.go like @prabushyam mentioned.